README.md

```diff
@@ -0,0 +1,42 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
```

Comment:
- [ ] [OPTIONAL] Please, consider changing the title for this project - I think "Portfolio" title came from previous projects 👍